dsl_parser/elements/node_templates.py

```diff
@@ -537,7 +537,7 @@ def parse(self, host_types, plugins):
             processed_nodes=processed_nodes,
             host_types=host_types,
             plugins=plugins)
-        return processed_nodes.values()
+        return list(processed_nodes.values())
```

Comment:
things like this are returned to who-knows-where, so let's keep backwards compat by still returning a list on py3